website/static/js/pages/base-page.js

```diff
@@ -123,4 +123,15 @@ $(function() {
             new KeenTracker(window.contextVars.keenProjectId, window.contextVars.keenWriteKey, params);
         }
     }
+
+    // Ad script alert
+    var adBlockPersistKey = 'adBlock';
+    var $adBlock = $('#adBlock').on('closed.bs.alert', function() {
```

Comment:
This code doesn't need to run on every page; base-page.js is included on every page. This should only be executed on the "Analytics" page. To do this, you can add a new `-page` JS module for the analytics page (statistics.mako). See http://cosdev.readthedocs.io/en/latest/osf/js_modules.html#building-and-using-modules